package.json

```diff
@@ -38,25 +39,50 @@
     "karma-jasmine": "1.1.0",
     "lodash": "4.17.4",
     "mousetrap": "1.4.6",
-    "neutrino": "4.3.1",
-    "neutrino-lint-base": "4.3.1",
-    "neutrino-preset-karma": "4.2.0",
-    "neutrino-preset-react": "4.2.3",
+    "neutrino": "^5.2.0",
+    "neutrino-middleware-copy": "^5.2.0",
+    "neutrino-middleware-eslint": "^5.2.0",
+    "neutrino-preset-karma": "^5.2.0",
+    "neutrino-preset-react": "^5.2.0",
     "ngreact": "0.3.0",
     "raw-loader": "0.5.1",
     "react": "15.3.1",
     "react-addons-test-utils": "15.3.1",
     "react-dom": "15.3.1",
     "taskcluster-client": "1.6.3",
+    "webpack": "^2.3.2",
     "webpack-md5-hash": "0.0.5"
   },
+  "neutrino": {
+    "use": [
+      "./neutrino-custom/lint.js",
+      "neutrino-preset-react",
+      "./neutrino-custom"
+    ],
+    "options": {
+      "source": "ui",
+      "output": "dist",
+      "entry": "entry-index.js",
+      "tests": "tests",
+      "compile": {
+        "targets": {
+          "browsers": [
+            "last 1 Chrome versions",
+            "last 1 Firefox versions",
+            "last 1 Edge versions",
+            "last 1 Safari versions"
+          ]
+        }
+      }
+    }
+  },
   "scripts": {
-    "build": "./node_modules/neutrino/bin/neutrino build --presets ./neutrino-custom/production.js",
-    "lint": "./node_modules/eslint/bin/eslint.js ui/",
-    "start": "./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "start:local": "./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/local-watch.js",
-    "start:stage": "SERVICE_DOMAIN=https://treeherder.allizom.org ./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "test": "./node_modules/neutrino/bin/neutrino test --presets ./neutrino-custom/test.js",
-    "test:watch": "./node_modules/neutrino/bin/neutrino test --watch --presets ./neutrino-custom/test.js"
+    "build": "./node_modules/neutrino/bin/neutrino build",
+    "lint": "NODE_ENV=development ./node_modules/eslint/bin/eslint.js --ext .js,.jsx ui/",
```

Comment:
Can `--ext .js,.jsx` be put in the config file instead, or does eslint not support that?

Also, at the moment we have the eslint version set in two places - the top level package.json entry, and via the neutrino dependency. It seems unfortunate to have to keep those in sync. But if we remove the top-level eslint entry, (a) we get a peer dependency warning (or at least we did with neutrino v4), (b) due to yarnpkg/yarn#2874 the bin script won't be created. Is there a better way to do this? Could neutrino add eslint to it's own package.json `bin` section per https://github.com/yarnpkg/yarn/issues/2874#issuecomment-290668434 ?